app/dashboard/src/pages/dashboard/Dashboard.tsx

```diff
@@ -145,6 +147,30 @@ function DashboardInner(props: DashboardProps) {
   const openProjectMutation = projectHooks.useOpenProjectMutation()
   const renameProjectMutation = projectHooks.useRenameProjectMutation()
 
+  useQuery({
```

Comment:
I think it's better to use `usePrefetchQuery` here: https://stackoverflow.com/a/77486960